examples/defaulter-gen/generators/defaulter.go

```diff
@@ -497,6 +498,24 @@ func mustEnforceDefault(t *types.Type, depth int, omitEmpty bool) (interface{},
 	}
 }
 
+var refRE = regexp.MustCompile(`^ref\((("(?P<pkg>.*)")\.)?(?P<ident>[^"]*)\)$`)
```

Comment:
I think you have an extraneous capture group.
What about this:
```suggestion
var refRE = regexp.MustCompile(`^ref\(((?P<pkg>.*)\.)?(?P<ident>[^"]*)\)$`)
```

Reply:
or something simpler like this since we don't really need the split:
```
var refRE = regexp.MustCompile(`^ref\((?P<reference>[^"])\)$`)
```